src/FlatButton/FlatButton.js

```diff
@@ -258,6 +263,7 @@ class FlatButton extends Component {
     return (
       <EnhancedButton
         {...other}
+        className={className}
```

Comment:
I am not able to understand your approach to handle className. className is not defined in EnhancedButton or its parent so according to my understanding we would have to explicitly need a className property here to make it work. Can you confirm my understanding or give an example to show me how you would make it work without explicitly having className property there.